components/automate-ui/src/app/components/chart-progress-bar/chart-progress-bar.component.ts

```diff
@@ -0,0 +1,123 @@
+import { Component, Input, OnInit, OnChanges } from '@angular/core';
+import { Styles } from './chart-progress-bar.model';
+
+@Component({
+  selector: 'app-chart-progress-bar',
+  templateUrl: './chart-progress-bar.component.html',
+  styleUrls: [ './chart-progress-bar.component.scss' ]
+})
+
+export class ChartProgressBarComponent implements OnInit, OnChanges {
+
+  @Input() size = 'large'; // large or small
+  @Input() value: number;
+  @Input() min: number;
+  @Input() max: number;
+  @Input() threshold: number[]; // [min, max]
+  @Input() valueTooltip: any;
+  @Input() maxTooltip: any;
```

Comment:
I think there are both the type of `number`?

Reply:
@SEAjamieD I'll update type, it's a string. 👍🏻